windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,103 @@
+
+// Addresses WINC-551 - https://github.com/openshift/windows-machine-config-operator/pull/279
+==== Added support for autoscaling Windows nodes
+
+After defining and deploying a ClusterAutoscaler, MachineSet, and a MachineAutoscaler, then
+// not sure what else goes here
+
+[id="wmco-3-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, The load balancer service would become unstable when you would scale additional Windows nodes. Now, the load balancer service has been stabilized to allow for adding multiple Windows nodes without erratic performance. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[**BZ#1905950**])
+
+* (link:https://bugzilla.redhat.com/show_bug.cgi?id=1919391[**BZ#1919391**])
+
+* (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924559[**BZ#1924559**])
+
+* Previously, when you replaced a cloud-private-key secret with a new private key, the WMCO patched pub-key-has with an annotation to the Linux node as well as the master and worker node. Now, when you change this private key, no annotation is added. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930791[**BZ#1930791**])
+
+* Previously, the kube-proxy service would terminate unexpectedly after the load balancer is created if you created the load balancer after the Windows pods begin running. Now, the kube-proxy service does not crash when recreating the load balancer service. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[**BZ#1939968**])
+
+* Previously, the `windows_exporter` component reported various metrics as `windows_*`. This error caused some of the node-level metrics that provide insight into the nodes through telemetry services to go unreported. Now, the componets export correctly showing all of the expected metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1948037[**BZ#1948037**])
+
+* Previously, empty IP address values in the load balancer's ingress would break the data path and the Windows service would be unreachable. This issue was due to the kube-proxy component considering empty IP address values in the ingress as invalid. Now, the Windows service is reachable even if the IP address value is empty. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**])
+
+* Previously, when the WMCO configured Windows instances, if the hybrid-overlay or kube-proxy components failed, the node might report itself as `Ready`. Now, the error is detected and the node reports itself as `NotReady`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1971745[**BZ#1971745**])
+
+* Previously, when you created a Windows pod with a projected volume, the pod would remain stuck in the "ContainerCreating" phase. Now, the Windows pod creation successfully proceeds to the "Running" phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953692[**BZ#1953692**])
+
+=== RHSA-2021:76465-04 - Windows Container support for {product-title} security update
+
+[id="wmco-3-0-0-known-issues"]
+=== Known issues
```

Comment:
Consider removing empty sections

Reply:
We may need to discuss this and add to it.

Reply:
This is still a work in progress in my view, so if we have no known fixes we can remove later.